Base/Python/slicer/parameterNodeWrapper/guiConnectors.py

```diff
@@ -129,7 +129,8 @@ def createGuiConnector(widget, datatype) -> GuiConnector:
     for possibleConnectorType in _registeredGuiConnectors:
         if possibleConnectorType.canRepresent(widget, datatype):
             return possibleConnectorType.create(widget, datatype)
-    raise RuntimeError(f"Unable to create GUI connector from datatype '{datatype}' to widget type '{type(widget)}'")
+    raise RuntimeError(f"Unable to create GUI connector from datatype '{datatype}' to widget type '{type(widget)}'. "
```

Comment:
What do you think of doing something like this to make the message a bit more helpful?

```python
import inspect
guiConnectorFilePaths = set()
for guiConnector in slicer.parameterNodeWrapper.guiConnectors._registeredGuiConnectors:
    guiConnectorFilePaths.add(inspect.getfile(guiConnector))

raise RuntimeError(f"Unable to create GUI connector from datatype '{datatype}' to widget type '{type(widget)}'."
                    " Please check the corresponding canRepresent function for clues about supported classes."
                    " To determine which data types can be connected to which widget types, please check 'canRepresent' methods in 'GuiConnector' classes"
                    " in these files: {', '.join(guiConnectorFilePaths)}")
```

Reply:
Better, gives more information. Do you want me to run this and see how the output looks?

Reply:
yes, please

Reply:
This is the output:
```
Traceback (most recent call last):
  File "c:/d/_Extensions/SlicerMorph_R/lib/Slicer-5.7/qt-scripted-modules/UberonTerminologyImporter.py", line 103, in setup
    self.initializeParameterNode()
  File "c:/d/_Extensions/SlicerMorph_R/lib/Slicer-5.7/qt-scripted-modules/UberonTerminologyImporter.py", line 138, in initializeParameterNode
    self.setParameterNode(self.logic.getParameterNode())
  File "c:/d/_Extensions/SlicerMorph_R/lib/Slicer-5.7/qt-scripted-modules/UberonTerminologyImporter.py", line 158, in setParameterNode
    self._parameterNodeGuiTag = self._parameterNode.connectGui(self.ui)
  File "C:\d\S5R\Slicer-build\bin\Python\slicer\parameterNodeWrapper\wrapper.py", line 253, in _connectGui
    _connectParametersToGui(self, paramNameToWidget)
  File "C:\d\S5R\Slicer-build\bin\Python\slicer\parameterNodeWrapper\wrapper.py", line 199, in _connectParametersToGui
    mappingToConnector = {
  File "C:\d\S5R\Slicer-build\bin\Python\slicer\parameterNodeWrapper\wrapper.py", line 200, in <dictcomp>
    paramName: value if isinstance(value, GuiConnector) else createGuiConnector(value, self.dataType(paramName))
  File "C:\d\S5R\Slicer-build\bin\Python\slicer\parameterNodeWrapper\guiConnectors.py", line 139, in createGuiConnector
    raise RuntimeError(f"Unable to create GUI connector from datatype '{datatype}' to widget type '{type(widget)}'."
RuntimeError: Unable to create GUI connector from datatype '<class 'str'>' to widget type '<class 'PythonQt.CTKWidgets.ctkPathLineEdit'>'. To determine which data types can be connected to which widget types, please check 'canRepresent' methods in 'GuiConnector' classes in these files: C:\d\S5R\Slicer-build\bin\Python\slicer\parameterNodeWrapper\guiConnectors.py, C:\d\S5R\Slicer-build\lib\Slicer-5.7\qt-scripted-modules\SubjectHierarchyLib\parameterNodeWrapper\guiConnectors.py
```

I removed the second line because it seemed superfluous and had to add an `f` postfix for the last line (apparently the first `f` does not get applied to subsequent lines).

Reply:
It looks good to me so I updated the PR with these changes.